packages/frontend-utils/src/api-util/api-util.ts

```diff
@@ -10,10 +9,9 @@ export type GetListOptions = {
 };
 
 export const createListApiUrl = (
-  endpoint: string,
+  url: URL,
```

Comment:
I think by shifting the constructing of the URL to the client of this util function you are removing some of it's functionality too, I realise it's because the package does not have access to the config - perhaps you considered making this a factory and then importing in FE, injecting and exporting again with the base url already in it?